charts/ingress-nginx/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 2.4.0
+version: 2.5.0
```

Comment:
```suggestion
version: 2.6.0
```

Another change got in before yours. Please bump the chart version and squash the commit. After that this lgtm

Reply:
done

Reply:
Sorry about this. Yet another change was merged in before yours again. Gonna need to bump the minor version and rebase your changes.

Reply:
do you mean chart version 2.6.1 ?

Reply:
So we should bump to `2.7.0`

Kubernetes follows [semver](https://semver.org/), [k8s docs](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions). So Major.Minor.Patch

Reply:
gotcha, done